cumulus/polkadot-parachain/src/chain_spec/bridge_hubs.rs

```diff
@@ -24,21 +24,14 @@ use std::str::FromStr;
 /// Collects all supported BridgeHub configurations
 #[derive(Debug, PartialEq)]
 pub enum BridgeHubRuntimeType {
+	Kusama,
+	Polkadot,
```

Comment:
```suggestion
```

this should go away also

Reply:
hmm, I dont know, I probably missed the discussion, so chainspecs for Kusama/Polkadot-like parachains are staying here? e.g. `./../chain-specs/bridge-hub-polkadot.json` ?